content/blog/lifehacking/2017-12-01_freedom_and_productivity.md

```diff
@@ -0,0 +1,199 @@
+productive means contributing to growth thus the good economic health of a
+entity greater than yourself (your country, your region, the world). And since
+people mix up being productive with working long hours, the confusion is easy
+to make. But I think productivity is the opposite of this: trying to work the
+least number of hours to advance work on the most important and valuable
+things. Begin maximally efficient with the minimum amount of resources.
+Identify those actions which will have the most positive impact, help the
+greatest number of people, be *valuable*. So it is another time to acknowledge
+the need for a why: why do you work for, in the first place? Is the work you're
+doing valuable, or are you doing busy work (like handling emails, answering
+very quickly on instant chat systems, etc.) to make it look like you're
+productive? Or more humbly, one might just be working to get the regular pay
+check, since work doesn't have to be a source of fulfillment, but instead can
+just be a means to an end too. Again, all motives are good here; but there
+should be one to make sense out of it.
+
+## To do what you want, to want what you do
+
+I think that's one goal of productivity: to be more conscious of all the
+choices we're making (what to work on) so that our work is more in adequation
+with our principles and real objectives. Thus it is about freedom, that is, the
+way to express what you truly desire to do and then to do it with purpose. Or
+in the words of French writer and journalist Françoise Giroud: happiness is
+doing what you want, and wanting what you do. I think other people call this
+autonomy or even integrity; I hope you get the idea.
+
+In particular, I don't want others to preconceive life choices for myself. Even
+if they have theoritized the topic and proven with objective metrics it worked
+for them. So here is my meta-productivity advice: to try different methods,
+and to make your own solutions out of them, to choose what works according to
+the context. Even if I need to spend a lot of time on thinking things through
+first, before applying my own crafted solutions, it is such a lower entry cost
+than doing erratic stuff without purpose and realize I lost my time. Of course,
+I might not find the best solution on the first try, but that's alright. It
+might mean I switch routines from one week to the other, or I don't have a
+consistent agenda; that is okay. I can experiment with many possible ways to do
+one thing, according to my current context: my state of mind, geographical
+location, needs, expectations, levels of energy, unexpected events popping up
+throughout the day, political context, really anything. As long as everything I
+do is conscious and works for me and makes me feel more fulfilled, I'll do the
+tradeoff of not having perfect control and a predefined answer to everything.
+
+What some call the review process matters here: one has to make sure they're
```

Comment:
Nit: Readers with no CS background may find "peer review process" more transparent than "review process" (like, the process of putting stars on Amazon products?)

EDIT: Ah, some sort of self-review. I'm not sure about my nit anymore. You can ignore it. 😋